### PR TITLE
Webpack completed

### PR DESCRIPTION
**Pull request by carina-lxx**
@j-scola @Rdecoster 

Webpack set up completed for review.

### CODE REVIEW

**Comment by j-scola** [comment]
I think there is a typo in the webpack config?? I think line 4 should be "mode" not node. I can't run the webpack script locally on my machine but still trying to figure out why.

**Comment by carina-lxx** [comment]
@j-scola  Hi James, I pushed one more time for the webpack. please review the diff.

**Review by j-scola — APPROVED** [review]
@carina-lxx verified webpack setup!

**Comment by j-scola** [comment]
@carina-lxx I noticed that `eslint --fix` is not yet working, this may be something to work on when you have an opportunity

File: webpack.config.js [hunk 1]
```diff
@@ -1,10 +1,21 @@
-const path = require('path');
-
-module.expots = {
-    node: 'development',
-    entry: './client/index.js',
+module.exports = {
```
Comment (j-scola) [inline webpack.config.js:1]: the webpack config file looks correct to me. good job

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
```
Comment (j-scola) [inline .eslintrc.json:4]: when I tried to run eslint, this line triggered an error `"es2021": true,`

Comment (carina-lxx) [reply .eslintrc.json:4]: I'm not sure about the eslint, I did start it in the terminal but seems like not working. I can still use "console.log"

**Pull request merged**